aiida/engine/processes/control.py

```diff
@@ -93,7 +93,7 @@ def play_processes(
     :raises ``ProcessTimeoutException``: If the processes do not respond within the timeout.
     """
     if not get_daemon_client().is_daemon_running:
-        raise DaemonException('The daemon is not running.')
+        LOGGER.warning('The daemon is not running, so process may not be reachable.')
```

Comment:
```suggestion
        LOGGER.warning('The daemon is not running, so some processes may not be reachable.')
```

Are there processes that _are_ reachable? Which ones?

Reply:
That's the thing: there _could_ be. For a process to be reachable, it just has to be active with "some" runner that is connected to RabbitMQ and is working on the corresponding task. The problem is that the daemon is just one possible way to have such a runner. So _it_ not running, is no guarantee that there isn't another runner working on the process and so being able to respond to RPCs. So we cannot have it except but still need to continue. But given that most users will only be using the main daemon, it not running essentially means that all processes will not be reachable.

Reply:
But, are there use cases where you are contollering AiiDA processes with a runner outside of the AiiDA daemon?

Reply:
Yes, for example when you "run" a process in an interactive shell, or if you run the daemon manually in the foreground with `verdi daemon worker`

Reply:
Ah, of course. These are still going through a runner. Thanks @sphuber!